Packs/FeedInfobloxThreatIntelligence/Integrations/FeedInfobloxThreatIntelligence/FeedInfobloxThreatIntelligence.yml

```
@@ -0,0 +1,400 @@
+- section: Collect
+  display: Max Indicators Per Fetch
+  name: max_fetch
+  type: 0
+  required: false
+  defaultvalue: "1000"
+  additionalinfo: "The maximum number of indicators to fetch in each run.\n\nNote: The maximum allowed value is 50000."
+- section: Collect
+  display: DGA Threat
+  name: dga_threat
+  type: 15
+  required: false
+  options:
+  - 'Yes'
+  - 'No'
+  additionalinfo: Filter the indicators having threats originated from dynamically generated algorithms.
+- section: Collect
+  display: Threat Classes
+  name: threat_classes
+  type: 16
+  required: false
+  options:
+  - APT
+  - Bot
+  - CompromisedDomain
+  - CompromisedHost
+  - Cryptocurrency
+  - DDoS
+  - DNSTunnel
+  - ExploitKit
+  - ICS
+  - IllegalContent
+  - InternetInfrastructure
+  - IntrusionAttempt
+  - LimitedDistro
+  - Malicious
+  - MaliciousNameserver
+  - MalwareC2
+  - MalwareC2DGA
+  - MalwareDownload
+  - Parked
+  - Phishing
+  - Policy
+  - PolicyViolation
+  - Proxy
+  - Scam
+  - Sinkhole
+  - Spambot
+  - Suspicious
+  - UncategorizedThreat
+  - Undefined
+  - UnwantedContent
+  - WebAppAttack
+  - Whitelist
+  additionalinfo: Filters the indicators according to the selected threat classes.
+- section: Collect
+  display: Data Providers
+  name: data_provider_profiles
+  type: 16
+  required: false
+  options:
+  - IID
+  - AISCOMM
+  additionalinfo: Filter indicators by data provider profiles.
+- section: Collect
+  display: Indicator Reputation
+  name: feedReputation
+  type: 18
+  required: false
+  options:
+  - None
+  - Good
+  - Suspicious
+  - Bad
+  defaultvalue: Suspicious
+  additionalinfo: Indicators from this integration instance will be marked with this reputation.
+- section: Collect
+  display: Source Reliability
+  name: feedReliability
+  defaultvalue: B - Usually reliable
+  type: 15
+  required: true
+  options:
+  - A - Completely reliable
+  - B - Usually reliable
+  - C - Fairly reliable
+  - D - Not usually reliable
+  - E - Unreliable
+  - F - Reliability cannot be judged
+  additionalinfo: Reliability of the source providing the intelligence data.
+- section: Collect
+  display: Traffic Light Protocol Color
+  name: tlp_color
+  type: 15
+  required: false
+  options:
+  - RED
+  - AMBER
+  - GREEN
+  - WHITE
+  defaultvalue: AMBER
+  additionalinfo: The Traffic Light Protocol (TLP) designation to apply to indicators fetched from the feed.
+- section: Collect
+  display: Tags
+  name: feedTags
+  type: 0
+  required: false
+  additionalinfo: Supports CSV values.
+- section: Collect
+  display: Bypass exclusion list
+  name: feedBypassExclusionList
+  type: 8
+  required: false
+  additionalinfo: When selected, the exclusion list is ignored for indicators from this feed. This means that if an indicator from this feed is on the exclusion list, the indicator might still be added to the system.
+- section: Collect
+  display: ""
```

The `display` name for this parameter is empty. Please provide a user-friendly name, for example: `Indicator Expiration Policy`.

```suggestion
  display: Indicator Expiration Policy
```